src/Providers/StringMacroServiceProvider.php

```diff
@@ -16,11 +17,17 @@ public function register(): void
 
     public function boot(): void
     {
+        // This macro is depreciated in favor of Str::transformUrlsInvisibly().
+        // As soon as it is not being used anymore, it can be deleted.
         Str::macro('wrapUrlsInBlankATag', function ($str, $attributes = []): string {
             $wrapper = new UrlInStringWrapper($str);
             $wrapper->setAttributes($attributes);
 
             return $wrapper->getString();
         });
+
+        Str::macro('transformUrlsInvisibly', function ($str): string {
```

Comment:
the name does not really explain me what it does 🤔 maybe something like `transformDotsInUrls` or `transformDotsInUrlsInvisibly` would better explain its purpose at a glance?

Reply:
I like `transformDotsInUrlsInvisibly ` 👍  I've changed it!